.pre-commit-config.yaml

```diff
@@ -74,6 +74,13 @@ repos:
     hooks:
       - id: black
         exclude: ^(harness/determined/common/api/bindings.py)
+  - repo: https://github.com/pycqa/flake8
+    # Note: this should be kept in sync with the version in `requirements.txt`.
```

Comment:
Would setting it for the global requirements file not be enough? Presumably if flake is in PATH, it's the one from the requirements file.
I think we're more likely to forget to update both versions than we are to be bitten in the butt by a version mismatch.

Reply:
IIRC pre-commit installs and manages its own virtual env and dependencies

Reply:
also it's a required field:
```
An error has occurred: InvalidConfigError:
==> File .pre-commit-config.yaml
==> At Config()
==> At key: repos
==> At Repository(repo='https://github.com/pycqa/flake8')
=====> Missing required key: rev
```